### ISSUE TITLE
Typo inside documentation

### ISSUE THREAD

**Issue by MakMuftic** [centrifuge/go-substrate-rpc-client#105]
There is a small typo inside one of the examples ([MakeASimpleTransfer](https://pkg.go.dev/github.com/centrifuge/go-substrate-rpc-client#example-package-MakeASimpleTransfer)) in documentation.

There is a check for error existence even though there is no reason for this check.

```go
// Create the extrinsic
ext := types.NewExtrinsic(c)
if err != nil {
	panic(err)
}
```

**Comment by philipstanislaus** [comment]
Great findings, thanks! Would you be able to submit a PR for those issues? Many thanks!

**Comment by MakMuftic** [comment]
@philipstanislaus yes for sure, I will submit PR by the end of this week

**Comment by mikiquantum** [comment]
fixed by #111
